.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: Release
+
+on:
+  push:
+    branches: ["main"]
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  build:
+    name: Release
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - uses: pnpm/action-setup@v2.0.1
+        with:
+          version: 7.3.0
```

Comment:
Lets harmonize all these pnpm versions across these workflow files and Dockerfile to a single version. May be we should declare an org/repo env var for it?

Reply:
I've updated the versions to `7.4.0`. For now, I don't think Github actions support org/repo env (Or I can't seem to find it) Only GitHub secrets which I don't think a version should be.